### PR TITLE
Sarthak Srivastav made changes

### PR DESCRIPTION
**Pull request by SarthakSrivastav**
Please merge my 'temp' branch in your master branch and deploy it on your site, so that I feel more confident and move my next step towards opensource contribution.

### CODE REVIEW

**Review by kaiwalyakoparkar — CHANGES_REQUESTED** [review]
Maximum 3 skills should be added. Refer : https://github.com/codeforcauseorg/pledge/blob/main/hints/create_pledge.md . 3rd point

**Comment by SarthakSrivastav** [comment]
Hi Kaiwalya Koparkar,

**Comment by SarthakSrivastav** [comment]
Hey Kaiwalya now I modified my Skills data in the repo and successfully commit it, will you please review and merge it.

**Comment by SarthakSrivastav** [comment]
Hey Codeforcauseorg/Pledge, I made the required changes in the file will
you please review and merge it now.

**Review by kaiwalyakoparkar — APPROVED** [review]
LGTM

**Comment by kaiwalyakoparkar** [comment]
Hey Sarthak, I don't have the rights to merge the pull request.. but I have reviewed and approved your request

**Comment by SarthakSrivastav** [comment]
Okay, Thanks.

File: src/data/contributors/SarthakSrivastav.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (shreyshreyansh) [inline src/data/contributors/SarthakSrivastav.json:1]: Looks good to go

**Pull request closed without merging**